tests/command/toggleattributecommand.js

```diff
@@ -5,7 +5,7 @@
 
 import Editor from '/ckeditor5/core/editor/editor.js';
 import Document from '/ckeditor5/engine/model/document.js';
-import AttributeCommand from '/ckeditor5/core/command/attributecommand.js';
+import ToggleAttributeCommand from '/ckeditor5/core/command/toggleattributecommand.js';
```

Comment:
I renamed `AttributeCommand` to `ToggleAttributeCommand`, but now I'm thinking about `AttributeTogglerCommand`?